test/SlowTests/Server/Documents/ETL/Elasticsearch/ElasticsearchEtlTests.cs

```diff
@@ -0,0 +1,871 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Nest;
+using Newtonsoft.Json.Linq;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.ETL.Elasticsearch;
+using Raven.Client.ServerWide.Operations.Certificates;
+using Raven.Server.Documents.ETL.Providers.Elasticsearch;
+using Raven.Server.Documents.ETL.Providers.Elasticsearch.Test;
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.ServerWide.Context;
+using Raven.Tests.Core.Utils.Entities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Server.Documents.ETL.Elasticsearch
+{
+    public class ElasticsearchEtlTests : EtlTestBase
+    {
+        public ElasticsearchEtlTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        protected const string defaultScript = @"
+var orderData = {
```

Comment:
Does this make sense in terms of Elastic usage? This is the script we use for SQL, no?

Reply:
It is. I reused it for testing `Parent-Child` relations.